mesures_with_latency_migr.npf

```diff
@@ -7,26 +7,36 @@ n_runs=10
 timeout=150
 
 %variables
+
 DELAY={0,5,10,25,50}
 
+TUNNEL_PENALTY=1
+
+heatmap:DELAY=0
```

Comment:
in heatmap mode (I used that to describe the fact you wanted two axis) then DELAY is overwritten to 0 as it's unused. Else we would have 3 factors...